lib/smart_answer_flows/marriage-abroad/outcomes/outcome_os_commonwealth.govspeak.erb

```diff
@@ -43,11 +43,36 @@
     <% end %>
   <% when 'india' %>
 
-    The British High Commission in India can provide you with a letter explaining this position for a charge - see [consular fees for India](/government/publications/india-consular-fees).
+    Download and print the [information note on ‘no objection certificates’](https://www.gov.uk/government/publications/marriage-information-note-on-noc) and show it to the registrar.
+
+    $A
+    British High Commission New Delhi
+    Shantipath, Chanakyapuri
+    New Delhi 110021
+    India
+    $A
+
+    $C
+    General telephone: +91 (11) 2419 2100 (24 hrs a day)
+
+    Management fax: +91 (11) 2419 2492
+
+    Press & Public Affairs fax: +91 (11) 2419 2411
+
+    Economic & Commercial fax: +91 (11) 2419 2301
+
+    Defence fax: +91 (11) 2419 2252
+
+    Political fax: +91 (11) 2419 2464
+
+    Consular fax: +91 (11) 2419 2491 (not for visa issues)
+
+    Email: [web.newdelhi@fco.gov.uk](mailto:web.newdelhi@fco.gov.uk)
```

Comment:
@erikse  Change made in this [commit](https://github.com/alphagov/smart-answers/pull/2365/commits/78f0e00b43cc0244a0fa6ecb3ee86348df793f5b)